desktop-exporter/app/components/header.tsx

```diff
@@ -20,6 +20,7 @@ export function Header(props: HeaderProps) {
       className="header"
       bg={useColorModeValue("gray.100", "gray.900")}
       align="center"
+      h={"60px"}
```

Comment:
I think my preference here would be using the long version of [these style props](https://chakra-ui.com/docs/styled-system/style-props) for clarity, so this would be:

```jsx
height={60}
paddingX={2}
```

but I'll leave you to use whichever sparks joy for you

Reply:
I'm going to fix this in the next PR because... well, to be honest I already started, and then realized I forgot to switch branches.